src/api/v1/api.json

```diff
@@ -347,6 +347,84 @@
           "response": null
         }
       },
+      "/:id/visit_activities": {
+        "GET": {
+          "description": "Retrieve all visit activities for an organistion",
+          "isImplemented": true,
+          "auth": true,
+          "intendedFor": ["TWINE_ADMIN", "FUNDING_BODY", "ORG_ADMIN", "VISITOR"],
```

Comment:
So there's an issue with how these kinds of routes in the `api.json` get translated into role/permissions relations by `database/utils/permissions.js`. Roles like `VISITOR` end up with `xxxx-child:zzzz` permissions, which aren't appropriate (it should only have either `xxxx-own:zzzz` or `xxxx-parent:zzzz` depending on the resource), similar for `ORG_ADMIN`.

I "fixed" this in my PR by separating `/community-businesses/:id` routes (which have `child` scopes) from `/community-businesses/me` routes (which have `own` scopes). IMO we should do a similar thing here, which will prevent the wrong scopes being assigned to `ORG_ADMIN`, though I don't think it will solve the `VISITOR` case, since it will either use an `/:id` route or a `/me` route, and the `permissions.js` file currently has no way of differentiating which of the entries in the `scope` array should be assigned to which role.

Reply:
@eliascodes aren't we trashing the `VISITOR` scope?

Reply:
@eliascodes aren't we trashing the concept of `VISITOR` for now?

Reply:
The use of `VISITOR` was just an example, same goes for `ORG_ADMIN`

Reply:
This should work if we drop `VISITOR` and replace `:/id/` -> `/me`. This would leave `ORG_ADMIN` as the only user type to access `/me/` so it can have `xxxx-own:zzzz`.

`TWINE_ADMIN` and `FUNDING_BODY` will use the `/:id/` route with `xxx-child:zzzz` with the appropriate prerequisites.